tests/testStreamFramesProcessed.cpp

```diff
@@ -51,12 +51,16 @@ void StreamFramesProcessed::TearDown() {
 TEST_P(StreamFramesProcessed, VerifyFramesProcessed) {
     const Direction direction = std::get<0>(GetParam());
     const int32_t sampleRate = std::get<1>(GetParam());
+    const bool useSampleRateConversion = std::get<2>(GetParam());
```

Comment:
useOboeSampleRateConversion

Sample Rate Conversion will happen somewhere. The question is whether it is in Oboe or lower down.